qiskit/circuit/singleton.py

```diff
@@ -156,7 +156,7 @@ def _singleton_lookup_key(n=1, label=None):
 
 there will be two singleton instances instantiated.  One corresponds to ``n=1`` and ``label=None``,
 and the other to ``n=2`` and ``label="two"``.  Whenever ``MySingleton`` is constructed with
-arguments consistent with one of those two cases, the relavent singleton will be returned.  For
+arguments consistent with one of those two cases, the relevent singleton will be returned.  For
```

Comment:
The correct spelling is actually "relevant":

```suggestion
arguments consistent with one of those two cases, the relevant singleton will be returned.  For
```